corehq/apps/app_manager/templates/app_manager/managed_app.html

```diff
@@ -119,10 +119,15 @@
         {% endif %}
         {% if show_report_modules %}
           <div class="pull-left">
-            <button type="button" class="popover-additem-option new-module" data-type="report">
-              <i class="fa fa-bar-chart"></i> {% trans "Report Menu" %}
-              <p>{% blocktrans %}Worker performance.{% endblocktrans %}</p>
-            </button>
+              <button type="button" class="popover-additem-option new-module" data-type="report" {% if disable_report_modules %} disabled {% endif %}>
+                  <i class="fa {% if disable_report_modules %}fa-ban{% else %}fa-bar-chart{% endif %}"></i>
+                  {% trans "Report Menu" %}
+                  {% if disable_report_modules %}
+                      <p>{% blocktrans %}Mobile reports are disabled because this is a linked domain.{% endblocktrans %}</p>
```

Comment:
Do we want to tell the users that this can be done with developer's intervention or by reaching out to product and would just need more support/$?

Reply:
I don't think so, but I'll check with the Solutions product team. I can change the text in a followup if necessary.